cpp/src/graphar/arrow/chunk_writer.cc

```diff
@@ -251,6 +252,25 @@ Status VertexPropertyWriter::WriteChunk(
   return Status::OK();
 }
 
+Status VertexPropertyWriter::WriteLabelChunk(
+    const std::shared_ptr<arrow::Table>& input_table,
+    IdType chunk_index, FileType file_type,
+    ValidateLevel validate_level) const {
+  auto schema = input_table->schema();
+  int indice = schema->GetFieldIndex(GeneralParams::kVertexIndexCol);
+  std::vector<int> indices;
+  for(int i=0; i<schema->num_fields(); i++){
```

Comment:
Is this would select all column?

Reply:
yes, as the input is label chunks, but here exists a bug: we do not need line 260